packages/ui/src/molecules/Alert/Alert.tsx

```diff
@@ -0,0 +1,73 @@
+import type { HTMLAttributes, MouseEvent, ReactNode } from 'react'
+import React, { forwardRef } from 'react'
+
+import Icon from '../../atoms/Icon'
+import Button from '../../atoms/Button'
+
+export interface AlertProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'onClose' | 'role'> {
```

Comment:
Gotta learn what this whole line means when I'm further on TS.
🥸

Reply:
To sum up, we are creating a new interface that extends the [HTMLAttributes](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Global_attributes) for _div_, but omitting the _onClose_ and _role_ attributes because we'd like to implement our own.

Reply:
I see, thank you!